notebooks/book1/02/softmax_plot.ipynb

```diff
@@ -1,55 +1,203 @@
 {
```

Comment:
<p>I think to avoid conditions written in lines 11-15, you can simply  modify line 3 as <code>fig, axs = plt.subplots(1, len(T),</code>sharey=True<code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/915/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>